src/main/java/com/seriouscompany/business/java/fizzbuzz/packagenamingpackage/interfaces/resources/ResourceProvider.java

```diff
@@ -0,0 +1,8 @@
+package com.seriouscompany.business.java.fizzbuzz.packagenamingpackage.interfaces.resources;
+
+public interface ResourceProvider {
```

Comment:
This is not clear at all what kind of Resources this ResourceProvider is providing, I think it should be called FizzBuzzInternationalizationStringResourceProvider for improved clarity.